packages/keymaps/README.md

```diff
@@ -20,7 +20,7 @@ Example of a valid `keymaps.json` file
 
 For most keys you can directly use the name of the key i.e `a`, `3`,  `/`, `-`.
 
-You can use `shift`, `ctrl`, `alt`, `meta`, `option` (`alt`), `command` (`meta`), `cmd` (`meta`) as modifiers. You can also `ControlOrCommand` or `CommandOrControl` as modifiers if you want the shortcut to fallback on `ctrl` coming from macOS. Note that if you defined a custom shortcut with `cmd`, `command` or `meta`, the same keymaps file won't work on a Windows/Linux machine as this key doesn't have an equivalent.
+You can use `shift`, `ctrl`, `alt`, `meta`, `option` (`alt`), `command` (`meta`), `cmd` (`meta`) as modifiers. You can also `ctrlcmd` as modifier if you want the shortcut to be `cmd` on OSX and `ctrl` on Windows/Linux. Note that if you defined a custom shortcut with `cmd`, `command` or `meta`, the same keymaps file won't work on a Windows/Linux machine as this key doesn't have an equivalent.
```

Comment:
It should be really clear that people should use `ctrlcmd` when they want `ctrl`. So it would be good to somehow say that first.

Reply:
Well I say: ` You can also `ctrlcmd` as modifier if you want the shortcut to be `cmd` on OSX and `ctrl` on Windows/Linux` 
I'm not sure that's it's true that they should write ctrlcmd if they want ctrl since on the mac that will become cmd...

Reply:
The 98% case is that all 'ctrl' shortcuts on win/linux translate to 'cmd' on mac. So using 'ctrl' is usually wrong.

Reply:
I'll try to clarify that.